CONTRIBUTING.md

```diff
@@ -177,7 +176,7 @@ If you want to user another DockerHub repository you need to set `DOCKER_HUB_REP
   - `GOVC_URL` same as `ESX IP`
   - `GOVC_USERNAME` & `GOVC_PASSWORD`: user credentials logging in to `ESX IP`
 
-- You **need** to set following environment variables to run swarm cluster related tests.
+- You **need** to set following environment variables to run swarm cluster related tests. You **need** to configure swarm cluster in order to run swarm related testcase otherwise there will be a test failure.
```

Comment:
Oh, so now I need 3 VMS **and** a swarm cluster? Or do I need to configure these VMs into a cluster? 
Please provide instructions or reference to configure swarm too.